lib/checks/media/description.js

```diff
@@ -1,12 +1,12 @@
-var tracks = node.querySelectorAll('track');
+var tracks = axe.utils.querySelectorAll(virtualNode, 'track');
+
 if (tracks.length) {
-	for (var i=0; i<tracks.length; i++) {
-		var kind = tracks[i].getAttribute('kind');
-		if (kind && kind === 'descriptions') {
-			// only return for matching track, in case there are multiple
-			return false;
-		}
-	}
-	return true;
+  // return false if any track has kind === 'description'
+  var out = !tracks.some(({ actualNode }) => (
+    actualNode.getAttribute('kind').toLowerCase() === 'descriptions'
```

Comment:
Could you add tests for both checks that ensures that th check returns without an exception when the markup looks like

```
<audio><track></track></audio>
```

i.e. no `kind` attribute at all